addons/test_mail_full/tests/test_ui.py

```diff
@@ -69,6 +69,15 @@ def test_message_actions_without_login(self):
             "message_actions_tour",
         )
 
+    def test_message_actions_portal_when_follower(self):
+        portal_user = self._create_portal_user()
+        self.record_portal.message_subscribe(partner_ids=[portal_user.partner_id.id])
+        self.start_tour(
+            f"/my/test_portal_records/{self.record_portal.id}?token={self.record_portal._portal_ensure_token()}",
+            "message_actions_tour",
+            login=portal_user.login,
+        )
```

Comment:
Do we really need to start a full tour? Can't we just call the route `/portal/chatter_init` in this case?